main/helpers.py

```diff
@@ -1,23 +1,49 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[3]:
+
+
 import pandas as pd
 import plotly.offline as py
 import plotly.graph_objs as go
 from .models import Tweet
 from django.utils.translation import gettext_lazy as _
 
 
-def df_from_tweets(only_symptoms=False):
+def df_all_tweets():
     dates = []
-    if only_symptoms == True:
-        iterator = Tweet.objects.filter(tweetannotation__symptom='yes')
-    else:
-        iterator = Tweet.objects.all()
+    iterator = Tweet.objects.all()
     for i in iterator:
         dates.append(i.date)
     df = pd.DataFrame(data={
         'date': dates,
         'has_symptom': dates})
     df['date'] = pd.to_datetime(df['date'])
-    adf = df.groupby(['date']).count()
+    df = df.groupby(['date']).count().reset_index()
+    return df
+   
+
+def compute_positive_rate():
+    df = pd.DataFrame.from_records(TweetAnnotation.objects.all().                                    values_list('tweet__tweet_id','tweet__date','symptom','uuid','created'),
+                                    columns=['tweet_id','date','symptom','uuid','timestamp'])
+    df=df[(df['symptom'].notna()) & (df['uuid'].notna())]
+    ids_yes = ids_yes.groupby(['tweet_id','date','symptom']).count()
+    ids_yes=pd.DataFrame(ids_yes['uuid'] / ids_yes.groupby('tweet_id')['uuid'].transform('sum')).reset_index()
+    ids_yes=ids_yes.loc[(ids_yes['symptom']=='yes')&(ids_yes['uuid']>=0.5),'id_str']
+    ids_yes=ids_yes.groupby('date').count().reset_index().rename(columns={'uuid':'nb_yes'})
+    df=df.groupby('date').count().reset_index().rename(columns={'uuid':'nb_annotations'})
+    df=pd.merge(df,ids_yes, on='date',how='outer')
+    df['percent_yes']=df['nb_yes']/df['nb_annotations']
+    return df
+    
+def df_from_tweets(only_sympotms=False):
```

Comment:
currently errors with 

```
6:43:15 PM web.1 |  Internal Server Error: /graph
6:43:15 PM web.1 |  Traceback (most recent call last):
6:43:15 PM web.1 |    File "/Users/bastian/.local/share/virtualenvs/twitterannotator-svV67Mjt/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
6:43:15 PM web.1 |      response = get_response(request)
6:43:15 PM web.1 |    File "/Users/bastian/.local/share/virtualenvs/twitterannotator-svV67Mjt/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
6:43:15 PM web.1 |      response = self.process_exception_by_middleware(e, request)
6:43:15 PM web.1 |    File "/Users/bastian/.local/share/virtualenvs/twitterannotator-svV67Mjt/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
6:43:15 PM web.1 |      response = wrapped_callback(request, *callback_args, **callback_kwargs)
6:43:15 PM web.1 |    File "/Users/bastian/dev/twitterannotator/main/views.py", line 66, in graph
6:43:15 PM web.1 |      graph = create_graph()
6:43:15 PM web.1 |    File "/Users/bastian/dev/twitterannotator/main/helpers.py", line 59, in create_graph
6:43:15 PM web.1 |      all_tweets_df = df_from_tweets()
6:43:15 PM web.1 |    File "/Users/bastian/dev/twitterannotator/main/helpers.py", line 41, in df_from_tweets
6:43:15 PM web.1 |      if only_symptoms == True:
6:43:15 PM web.1 |  NameError: name 'only_symptoms' is not defined
```

because of the typo in the function call :D